polysemy-plugin/src/Polysemy/Plugin.hs

```diff
@@ -100,12 +94,9 @@ installTodos todos = do
     0 -> pure todos
     _ -> do
       mods <- moduleSetElts <$> getVisibleOrphanMods
-      pure $ case any ((== polysemyInternal) . moduleName) mods of
-        True  -> CoreDoPluginPass "Inline Recursive Calls" inlineRecursiveCalls
-               : todos
+      pure $ todos
```

Comment:
This change makes `mods` unused in the < `810` case, right? That'll make lint unhappy

Reply:
Good point, thanks.